upt_macports/upt_macports.py

```diff
@@ -2,15 +2,16 @@
 import logging
 import jinja2
 
+
 class MacPortsPackage(object):
     def __init__(self):
         self.logger = logging.getLogger('upt')
 
     def create_package(self, upt_pkg, output):
         self.upt_pkg = upt_pkg
-        #self._create_output_directories(upt_pkg, output)
-        #self._create_makefile()
-        #self._create_pkg_descr()
+        # self._create_output_directories(upt_pkg, output)
+        # self._create_makefile()
+        # self._create_pkg_descr()
```

Comment:
If we are going to clean up, remove these lines.

Reply:
I have applied these changes and other suggested below, sorry for the silly mistakes :)